pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/ModelParser.groovy

```diff
@@ -321,7 +321,7 @@ class ModelParser implements Parser {
 
             // Lazily evaluate prettyPrint(...) - i.e., only if AST_DEBUG_LOGGING is true.
             astDebugLog {
-                "Transformed runtime AST:\n${ -> prettyPrint(pipelineBlock.whole.arguments)}"
+                "Transformed runtime AST:\n${ -> prettyPrint(src)}"
```

Comment:
Perhaps you could add a test where astDebugLog is enabled to protect it from future breakage?

Reply:
Good idea - I'll see what I can do.

Reply:
This is impressively difficult to really test - it needs a `ModuleNode` passed to it, specifically one that's been generated by `RuntimeASTTransformer` from a `SourceUnit` and a parsed `ModelASTPipelineDef` so that it'll actually look like the real thing. Lemme think on this...

Reply:
...doesn't help that for some reason I can't run tests from intellij right now. heh.

Reply:
woo, got it.